coral/src/app/layout/header/HeaderMenuLink.test.tsx

```diff
@@ -1,61 +1,101 @@
-jest.mock("@aivenio/design-system", () => {
+jest.mock("@aivenio/aquarium", () => {
   return {
     __esModule: true,
-    ...jest.requireActual("@aivenio/design-system"),
+    ...jest.requireActual("@aivenio/aquarium"),
 
     Icon: () => {
       return <div data-testid={"ds-icon"}></div>;
     },
   };
 });
 
+const linkText = "Go to your profile page";
+
 describe("HeaderMenuLink.tsx", () => {
   // (icon is not needed for the test, Icon component mocked out)
   const mockIcon = "" as unknown as typeof data;
 
   describe("renders a default link with required props", () => {
-    beforeAll(() => {
+    beforeEach(() => {
       render(
         <HeaderMenuLink
           icon={mockIcon}
           href={"/myProfile"}
-          linkText={"Go to your profile page"}
+          linkText={linkText}
         />
       );
     });
 
-    afterAll(cleanup);
+    afterEach(cleanup);
 
     it(`renders a link with text dependent on a given property`, () => {
       const navLink = screen.getByRole("link", {
-        name: "Go to your profile page",
+        name: linkText,
       });
 
       expect(navLink).toBeVisible();
     });
 
-    it(`renders a href for that link dependent on a given  property`, () => {
+    it(`renders a href for that link dependent on a given property`, () => {
       const navLink = screen.getByRole("link", {
-        name: "Go to your profile page",
+        name: linkText,
       });
       expect(navLink).toHaveAttribute("href", "/myProfile");
     });
 
-    it(`renders a Tooltip with an Icon, both hidden for assistive technology`, () => {
+    it(`renders an Icon`, async () => {
+      const icon = screen.getByTestId("ds-icon");
+
+      expect(icon).toBeVisible();
+    });
+
+    it(`triggers the rendering of a Tooltip on mouse hover`, async () => {
       const navLink = screen.getByRole("link", {
-        name: "Go to your profile page",
+        name: linkText,
       });
-      const tooltip = within(navLink).getByTestId("tooltip");
-      const icon = within(tooltip).getByTestId("ds-icon");
+      const queryForToolTip = () => screen.queryByRole("tooltip");
+
+      expect(queryForToolTip()).toBeNull();
+
+      await userEvent.hover(navLink);
+
+      expect(queryForToolTip()).toBeVisible();
+      expect(queryForToolTip()).toHaveTextContent(linkText);
+
+      // Assert the tooltip disappears after hover event stops
+      cleanup();
+      expect(queryForToolTip()).toBeNull();
+    });
+
+    it(`triggers the rendering of a Tooltip on tab navigation`, async () => {
+      const navLink = screen.getByRole("link", {
+        name: linkText,
+      });
+      const queryForToolTip = () => screen.queryByRole("tooltip");
+
+      expect(queryForToolTip()).toBeNull();
+
+      await tabNavigateTo({ targetElement: navLink });
+
+      expect(queryForToolTip()).toBeVisible();
+      expect(queryForToolTip()).toHaveTextContent(linkText);
+
+      // Assert the tooltip disappears after losing focus
+      cleanup();
```

Comment:
same as above!

Reply:
*Actually* these roles already are taken care of by the design system! So I reverted the commit.

And also, maybe they are not picked up by the macos screenreader? Or maybe their behaviour is not what I assume?


![Screenshot 2022-12-14 at 16 15 34](https://user-images.githubusercontent.com/20607294/207634695-bb526546-38d3-4dc5-b84f-8eb17160501f.png)